### PR TITLE
Feature/pom sonar update

### PR DESCRIPTION
**Pull request by wdhda-ch**
Update pom dependencies and config to improve sonar integration.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml [hunk 1]
```diff
@@ -241,142 +261,140 @@
...
+                            <goal>add-test-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>src/feature/java</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```
Comment (amartin7663) [inline pom.xml:397]: Do we need this integration test stage

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by amartin7663 — APPROVED** [review]

**Pull request merged**